apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -436,8 +436,11 @@ private ReadOrQueryRequest GetReadOrQueryRequest()
                     case SpannerCommandType.Read:
                         request = ReadOrQueryRequest.FromReadRequest(GetReadRequest());
                         break;
+                    case SpannerCommandType.Dml:
```

Comment:
This has the same code as the SpannerCommandType.Select case, so we can just add this case to that one, unless there's another change coming later.